CMakeLists.txt

```diff
@@ -339,7 +339,7 @@ file(READ ${PG_INCLUDEDIR}/pg_config.h PG_CONFIG_H)
 string(REGEX MATCH "#define USE_ASSERT_CHECKING 1" PG_USE_ASSERT_CHECKING ${PG_CONFIG_H})
 
 if (PG_USE_ASSERT_CHECKING AND NOT ASSERTIONS)
-  message(NOTICE "Assertion checks are OFF although enabled in PostgreSQL build (pg_config.h).\n"
+  message(WARNING "Assertion checks are OFF although enabled in PostgreSQL build (pg_config.h). "
```

Comment:
Please fix the other NOTICE as well.

Reply:
Switched to "(none)" (https://cmake.org/cmake/help/v3.11/command/message.html), which is supported by 3.11